docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -930,3 +930,53 @@ Then element located '$locator' disappears in '$timeout'
 When I click on a button with the name 'Toggle element visibility with delay'
 Then element located 'By.xpath(//div[@id='delayed'])' disappears in 'PT3S'
 ----
+
+== Tips & Tricks
+
+=== Validate URL host of the opened page
+
+To validate host component of the current page URL the following statement can be used:
+
+.Validate host component of current page URL
+
+.Validate host component of "https://example.com/relative/url/example.html"
+[source,gherkin]
+----
+Given I am on page with URL `https://example.com/relative/url/example.html`
+Then `#{extractHostFromUrl(${current-page-url})}` is equal to `example.com`
+----
+
+Where:
+
+* `${current-page-url}` is the <<Current page URL>> dynamic variable.
```

Comment:
we don't use "where"-approach to describe inputs
```suggestion
* `${current-page-url}` is the <<Current page URL>> dynamic variable.
```

Reply:
how is this line rendered in docs?

Reply:
the same for sections below

Reply:
![image](https://user-images.githubusercontent.com/48581960/233118514-b2a470f9-e1c6-47f2-8e17-13b755f1bd63.png)
And link leads to current page url title 
![image](https://user-images.githubusercontent.com/48581960/233118700-e1c0348b-1d1e-46ec-b8b1-b423b3dcf14f.png)

Reply:
no reason to capitalize `Current...`